setup/ansible-inventory

```diff
@@ -39,6 +39,9 @@ test-softlayer-ubuntu14-x64-1
 test-softlayer-ubuntu14-x86-1
 test-digitalocean-ubuntu14-x86-1
 test-digitalocean-ubuntu14-x64-1
+iojs-ns-xgene-1
+iojs-ns-xgene-2
+iojs-ns-xgene-3
 
```

Comment:
If you're rebaking these machines, could you perhaps use the new naming convention? In this case it would be
`test-nodesource-ubuntu14-arm-{1,2,3}`. Or is it perhaps `arm64`? (should match with what's passed to `--dest-cpu`).